crates/viewer/re_viewer/src/startup_options.rs

```diff
@@ -75,11 +75,27 @@ pub struct StartupOptions {
     /// viewers onto the same page, then it's better to turn this off.
     ///
     /// We use browser history in a limited way to track the currently
-    /// open example recording, see [`crate::history`].
+    /// open example or redap recording, see [`crate::history`].
     #[cfg(target_arch = "wasm32")]
     pub enable_history: bool,
 }
 
+impl StartupOptions {
+    /// Returns `StartupOptions::enable_history` on web, and `false` on native.
+    #[allow(clippy::unused_self)] // Only used on web.
+    pub fn enable_web_history(&self) -> bool {
```

Comment:
Nit: This reads like a verb, but doesn't take `&mut self`, which confused me for a bit.
```suggestion
    pub fn web_history_enabled(&self) -> bool {
```